frontend/webpack.config.ts

```diff
@@ -340,10 +340,8 @@ const config: Configuration = {
 if (CHECK_CYCLES === 'true') {
   new CircularDependencyPreset({
     exclude: /node_modules|public\/dist|\.(gql|html)$/,
-    // TODO: investigate how to load the plugins registry asynchronously
-    filterModules: /^get-active-plugins\.js$/,
     // TODO: Fix cycles unveiled by CONSOLE-4837, then remove the below line
-    thresholds: { totalCycles: 5 },
+    thresholds: { totalCycles: 310 },
```

Comment:
Removing `get-active-plugins` from `filterModules` to expose all import cycles here. This increased the number but the cycles were already there, just being suppressed.

Reply:
Resolved by #15688 #15689 #15708 #15718 #15723